scripts/get_qc_table.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import re
+import csv
+
+
+def parsing_cov(coverage_file):
+    with open(coverage_file, "r") as cov_file:
+        reader = csv.reader(cov_file, delimiter="\t")
+
+        for row in reader:
+            for element in row:
+                if not re.match("#", element):
+                    aligned_reads = row[3]
+                    total_cov = row[5]
+                    mean_depth = row[6]
```

Comment:
Those columns are defined by samtools. I'm parsing the samtools output here. 
But yes, after the experience with the vep-vcfs I see why it's probably better to read the header line

Reply:
Samtools is a good example for format changes.  The output format has changed a number of times in past releases, even between minor releases.  So it's really important to get the cells of interest from the header instead of trusting the order.